src/sass/components/Header.scss

```diff
@@ -0,0 +1,12 @@
+.Header {
```

Comment:
Feels like an odd fix for such a small thing ... and I'm not seeing that happen at all.